packages/navstar/build

```diff
@@ -5,7 +5,7 @@ source /opt/mesosphere/environment
 
 export CFLAGS="-I/opt/mesosphere/include -I/opt/mesosphere/active/libsodium/include"
 export LDFLAGS="-L/opt/mesosphere/lib -L/opt/mesosphere/active/libsodium/lib -Wl,-rpath=/opt/mesosphere/active/libsodium/lib"
-export CXXFLAGS=CFLAGS
+export CXXFLAGS=${CFLAGS}
```

Comment:
https://www.gnu.org/software/automake/manual/html_node/Flag-Variables-Ordering.html -- CPPFLAGS should be set if you want both CFLAGS and CXXFLAGS to be set